src/main/scala/gemmini/GemminiConfigs.scala

```diff
@@ -72,6 +73,84 @@ case class GemminiArrayConfig[T <: Data : Arithmetic, U <: Data](
   val mvout_len_bits = log2Up(meshColumns * tileColumns + 1)
   val mvout_rows_bits = log2Up(meshRows * tileRows + 1)
 
+  //==========================================================================
+  // sanity check mesh size
+  //==========================================================================
+  def BLOCK_ROWS = tileRows * meshRows
```

Comment:
Is the a convention for why these are capitalized, while the vals just before it are not?

Reply:
i use capitalized constants and lowercase signals to help the reader distinguish in the code.

Reply:
Why are all of these `def` instead of `val`? Do you really want to reevaluate them every time they are used?

Reply:
Constants are apparently supposed to be "upper cammel case": https://docs.scala-lang.org/style/naming-conventions.html.

But I don't think it's really important one way or the other.

Reply:
changed the defs to vals